doc/sphinx-guides/source/installation/oauth2.rst

```diff
@@ -1,4 +1,4 @@
-OAuth Login: ORCID, GitHub, Google
+OAuth Login: ORCID, Microsoft Azure Active Directory, GitHub, Google
 ==================================
```

Comment:
Whoops. Some "=" missing now.

Reply:
Thanks, corrected in https://github.com/IQSS/dataverse/pull/6192/commits/4931d6afaf2d58d4bca0b4255655c9d43076a882